test/integration/contract-aci.js

```diff
@@ -520,7 +520,7 @@ describe('Contract instance', function () {
 
       it('Invalid arguments length', async () => {
         await expect(testContract.methods.datTypeGFn())
-          .to.be.rejectedWith('Non matching number of arguments. Got 0 but expected from 1 to 1')
+          .to.be.rejectedWith('Non matching number of arguments. datTypeGFn expects between 1 and 1 number of arguments but got 0')
```

Comment:
Not sure where exactly the error is generated but the error message in here is confusing.

Reply:
agree

Reply:
This was changed on calldata lib side in https://github.com/aeternity/aepp-calldata-js/pull/110
Contract methods may accept a variable amount of arguments if the last arguments are optional. In this case, `datTypeGFn` accepts the one required argument, so we have "between 1 and 1".

Reply:
Oh, I see.